TUTORIAL.md

```diff
@@ -406,7 +411,7 @@ lakes or in the middle of a plain. Either way, you have established
 your base in the shade of what you assume is a tree and now can send
 out robots to explore!
 
-![World generated with seed 16](./images/world16.png)
+![World generated with seed 16](images/tutorial/world16.png)
```

Comment:
Should we also regenerate this image to have `?`s?

Reply:
Good point, I tried to kill two birds with one stone and used Creative mode that is in next section :slightly_smiling_face: